internal/archive/archive.go

```diff
@@ -77,12 +74,12 @@ func (a *ubuntuArchive) Options() *Options {
 	return &a.options
 }
 
-func (a *ubuntuArchive) Exists(pkg string) bool {
```

Comment:
This still looks like a nice part of the API. Assuming the suggested NotFoundError, we might implement it in terms of Version internally.